debian/changelog

```diff
@@ -1,3 +1,10 @@
+matrix-synapse-py3 (1.58.0+nmu1) UNRELEASED; urgency=medium
```

Comment:
Please check this version number is appropriate.

Reply:
don't sweat the specific version number here; the release process fixes this up to match whatever we are releasing.